SparkPost/2web2ui
# FE-1093 - Form field components should have a baked-in max-width

[FE-1093](https://jira.int.messagesystems.com/browse/FE-1093)

### What Changed

- Added wrapping `<Box/>` that can receive the `maxWidth` prop - defaulting to the `1200` design token
- Removed feature-by-feature setting of max width in favor of global component default prop
- Fixed a few spots in which the default max width needed an override

### How To Test

- Click through the app with Hibana enabled, visually verify forms
- Ensure most fields have the max width set
- Ensure there aren't unusual spots in which the max width should be set to something else!

### To Do
